src/lightning_app/core/work.py

```diff
@@ -398,10 +419,10 @@ def __getattr__(self, item):
             return path
         return self.__getattribute__(item)
 
-    def _call_hash(self, fn, args, kwargs):
+    def _call_hash(self, fn, args, kwargs) -> str:
         hash_args = args[1:] if len(args) > 0 and args[0] == self else args
         call_obj = {"args": hash_args, "kwargs": kwargs}
-        return f"{fn.__name__}:{DeepHash(call_obj)[call_obj]}"
+        return str(DeepHash(call_obj)[call_obj])[:7]
```

Comment:
Can you document what is expected here in terms of string content, since we are only retaining the first characters?